src/table/boot.rs

```diff
@@ -12,22 +14,33 @@ pub struct BootServices {
     header: Header,
 
     // Task Priority services
-    raise_tpl: extern "win64" fn(Tpl) -> Tpl,
-    restore_tpl: extern "win64" fn(Tpl),
+    raise_tpl: extern "win64" fn(new_tpl: Tpl) -> Tpl,
+    restore_tpl: extern "win64" fn(old_tpl: Tpl),
 
```

Comment:
Since I was around, I also took the opportunity to add some parameter names here for increased readability.